src/SFA.DAS.Campaign.Web/Views/LandingPages/ExpressionOfInterestThanks.cshtml

```diff
@@ -0,0 +1,9 @@
+﻿@{
+    ViewBag.Title = "";
```

Comment:
Do we need meta data values for this page?

Reply:
Probably, still awaiting content here. I created the empty file for now. I can take it out of the project?

Reply:
Yeah that's fine. Sounds like there's still some decisions to be made about this functionality. Gonna put a 'Do not merge' label on for now.